docs/building-modules.md

```diff
@@ -30,7 +30,13 @@ Hello universe!
 
 See [example modules](../examples) for more advanced examples.
 
-## Modules
+## Table of Contents
+
+- [Importing JavaScript Modules](#importing-javascript-modules)
+- [Platform APIs](#platform-apis)
+- [Testing Guide](#testing-guide)
```

Comment:
GitHub provides a built-in table of contents. However, I think may people are not aware of that feature. I am adding minimalistic ToCs here and there to make this page easier to navigate.

Reply:
Thanks for sharing, I wasn't aware of this either